test-tests/shared/src/test/scala/zio/test/BoolAlgebraSpec.scala

```diff
@@ -207,3 +198,15 @@ object BoolAlgebraSpec extends ZIOBaseSpec {
     if (a == b) (a, b) else randomEqualBoolAlgebra(size)
   }
 }
+
+object BoolAlgebraSpecHelper {
```

Comment:
Are you going to migrate this spec as part of this PR? If not I would leave this where it was.

Reply:
i think this will be last one for this pr

Reply:
Okay, great. That would be fantastic! Maybe call this `BoolAlgebraSpecUtil` to conform with the practice in `core` and `tests`?  Otherwise will review this file when it is done but everything else is good to go once the tests pass. It looks like there are some compilation issues right now. Let me know if you need any help in resolving.

Reply:
> Maybe call this BoolAlgebraSpecUtil to conform with the practice in core and tests?

actually named it like `ZIOSpec` companion `ZIOSpecHelper`

Reply:
Sounds good.